monitoring/grafana.py

```diff
@@ -2,10 +2,15 @@
 from sys import argv
 from datetime import datetime
 from calendar import timegm
+import configparser
 from bottle import (Bottle, HTTPResponse, run, request, response,
                     json_dumps as dumps)
+
+monitor_config = configparser.ConfigParser()
+monitor_config.read('monitoring.config')
```

Comment:
can you pass the path on the command line. reading from the current working directory can be problematic.

Reply:
The main piman config parser uses glob to resolve its config file, should we try that or should we stick with passing in the path to config as a command line argument?